sw/device/lib/testing/random.h

```diff
@@ -0,0 +1,30 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_RANDOM_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_RANDOM_H_
+
+#include <stdint.h>
+
+/**
+ * Generate a pseudo-random unsigned integer.
+ *
+ * For testing purposes, a simple LFSR generates random numbers starting with
+ * a known seed.
+ * @return A pseudo-random 32-bit value.
+ */
+uint32_t urandom(void);
+
+/**
+ * Generate a random unsigned integer between a given range.
```

Comment:
Is the range inclusive, or exclusive? The PRNG is clearly not a uniform distribution in general, so perhaps that should be noted here.

Reply:
Updated comments.